### DISCUSSION TITLE
1.19.3 doesn't work

### DISCUSSION THREAD

**Discussion by NBS-Paste** [FabricMC/fabric-api#2766] [Player support] [answered]
when i load into minecraft it crashes with this exxit code

-1

**Comment by warjort** [comment]
We can't help you if you don't show the crash report or logs/latest.log

**Comment by NBS-Paste** [comment]
[2022-12-19-2.log.gz](https://github.com/FabricMC/fabric/files/10261899/2022-12-19-2.log.gz)

**Comment by warjort** [comment]
````
[10:24:45] [Render thread/ERROR]: Mixin apply for mod continuity failed continuity.mixins.json:SpriteAtlasTextureMixin from mod continuity -> net.minecraft.class_1059: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow method method_18164 in continuity.mixins.json:SpriteAtlasTextureMixin from mod continuity was not located in the target class net.minecraft.class_1059. Using refmap continuity-refmap.json
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow method method_18164 in continuity.mixins.json:SpriteAtlasTextureMixin from mod continuity was not located in the target class net.minecraft.class_1059. Using refmap continuity-refmap.json
````
Looks like the version of continuity you have is not compatible with 1.19.3?
Or it could be conflicting with another mod.
Check you have the latest version then contact the mod author.

**Comment by NBS-Paste** [comment]
i took out the mod and it still wouldnt work

19:26:24.242
Configuration conflict: there is more than one oshi.architecture.properties file on the classpath: [jar:file:/C:/Users/Jack/AppData/Roaming/.minecraft/libraries/com/github/oshi/oshi-core/6.2.2/oshi-core-6.2.2.jar!/oshi.architecture.properties, jar:file:/C:/Users/Jack/AppData/Roaming/.minecraft/libraries/com/github/oshi/oshi-core/6.2.2/oshi-core-6.2.2.jar!/oshi.architecture.properties]
19:26:25.360

---- Minecraft Crash Report ----
// Why did you do that?

Time: 2022-12-19 19:26:24
Description: Initializing game

java.lang.AbstractMethodError: Receiver class net.devtech.arrp.impl.RuntimeResourcePackImpl does not define or inherit an implementation of the resolved method 'abstract net.minecraft.class_7367 method_14405(net.minecraft.class_3264, net.minecraft.class_2960)' of interface net.minecraft.class_3262.
	at net.minecraft.class_3294.method_14486(class_3294.java:72)
	at net.minecraft.class_6861.method_14486(class_6861.java:82)
	at foundationgames.enhancedblockentities.util.hacks.ResourceHacks.cropAndPutTexture(ResourceHacks.java:15)
	at foundationgames.enhancedblockentities.util.hacks.ResourceHacks.addChestParticleTexture(ResourceHacks.java:26)
	at foundationgames.enhancedblockentities.util.hacks.ExperimentalSetup.setupChests(ExperimentalSetup.java:51)
	at foundationgames.enhancedblockentities.util.hacks.ExperimentalSetup.setup(ExperimentalSetup.java:21)
	at net.minecraft.class_6861.handler$zzi000$enhanced_bes$injectExperimentalPack(class_6861.java:1043)
	at net.minecraft.class_6861.<init>(class_6861.java:60)
	at net.minecraft.class_3304.<init>(class_3304.java:30)
	at net.minecraft.class_310.<init>(class_310.java:517)
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_3294.method_14486(class_3294.java:72)
	at net.minecraft.class_6861.method_14486(class_6861.java:82)
	at foundationgames.enhancedblockentities.util.hacks.ResourceHacks.cropAndPutTexture(ResourceHacks.java:15)
	at foundationgames.enhancedblockentities.util.hacks.ResourceHacks.addChestParticleTexture(ResourceHacks.java:26)
	at foundationgames.enhancedblockentities.util.hacks.ExperimentalSetup.setupChests(ExperimentalSetup.java:51)
	at foundationgames.enhancedblockentities.util.hacks.ExperimentalSetup.setup(ExperimentalSetup.java:21)
	at net.minecraft.class_6861.handler$zzi000$enhanced_bes$injectExperimentalPack(class_6861.java:1043)
	at net.minecraft.class_6861.<init>(class_6861.java:60)
	at net.minecraft.class_3304.<init>(class_3304.java:30)
	at net.minecraft.class_310.<init>(class_310.java:517)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		AudioDevProps2.dll::2.6.18.0:A-Volute
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		CoreMessaging.dll:Microsoft CoreMessaging Dll:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		DEVOBJ.dll:Device Information Set DLL:10.0.22621.755 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		GLU32.dll:OpenGL Utility Library DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.22621.755 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.22621.900 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.22621.900 (WinBuild.160101.0800):Microsoft Corporation
		MSASN1.dll:ASN.1 Runtime APIs:10.0.22621.819 (WinBuild.160101.0800):Microsoft Corporation
		MSCTF.dll:MSCTF Server DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.2211.5 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		NahimicOSD.dll:NahimicOSD:2.2.26.0:Nahimic
		OLEAUT32.dll:OLEAUT32.DLL:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE for Windows:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		ProductInfo.dll::1.9.14.0:A-Volute
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		SETUPAPI.dll:Windows Setup API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.22621.755 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft� C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		WINTRUST.dll:Microsoft Trust Verification APIs:10.0.22621.900 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		cfgmgr32.dll:Configuration Manager DLL:10.0.22621.755 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		d3d9.dll:Direct3D 9 Runtime:10.0.22621.755 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:DHCP Client Service:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:DHCPv6 Client:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		dinput8.dll:Microsoft DirectInput:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		directxdatabasehelper.dll:DirectXDatabaseHelper:10.0.22621.963 (WinBuild.160101.0800):Microsoft Corporation
		dwmapi.dll:Microsoft Desktop Window Manager API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		dxcore.dll:DXCore:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		dxgi.dll:DirectX Graphics Infrastructure:10.0.22621.900 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.22621.963 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		icm32.dll:Microsoft Color Management Module (CMM):10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		igc64.dll:Intel Graphics Shader Compiler for Intel(R) Graphics Accelerator:31.0.101.3959:Intel Corporation
		igd12dxva64.dll:User Mode Driver for Intel(R) Graphics Technology:31.0.101.3959:Intel Corporation
		igdgmm64.dll:User Mode Driver for Intel(R) Graphics Technology:31.0.101.3959:Intel Corporation
		igdml64.dll:Metrics Library for Intel(R) Graphics Technology:31.0.101.3959:Intel Corporation
		igxelpicd64.dll:OpenGL(R) Driver for Intel(R) Graphics Accelerator:31.0.101.3959:Intel Corporation
		inputhost.dll:InputHost:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.3.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jna4253861770919701961.dll:JNA native library:6.1.4:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jvm.dll:OpenJDK 64-Bit server VM:17.0.3.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		lwjgl_opengl.dll
		lwjgl_stb.dll
		management.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		mscms.dll:Microsoft Color Matching System DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		msvcp140.dll:Microsoft� C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft� C Runtime Library:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:NT Layer DLL:10.0.22621.900 (WinBuild.160101.0800):Microsoft Corporation
		opengl32.dll:OpenGL Client DLL:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP Name Space Provider:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		powrprof.dll:Power Profile Helper DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		textinputframework.dll:"TextInputFramework.DYNLINK":10.0.22621.900 (WinBuild.160101.0800):Microsoft Corporation
		ucrtbase.dll:Microsoft� C Runtime Library:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		uxtheme.dll:Microsoft UxTheme Library:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft� C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		win32u.dll:Win32u:10.0.22621.963 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.22621.608 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows Base Types DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		xinput1_4.dll:Microsoft Common Controller API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.19.3
	Minecraft Version ID: 1.19.3
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1075149856 bytes (1025 MiB) / 1677721600 bytes (1600 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 12
	Processor Vendor: GenuineIntel
	Processor Name: 12th Gen Intel(R) Core(TM) i5-12400
	Identifier: Intel64 Family 6 Model 151 Stepping 5
	Microarchitecture: Alder Lake
	Frequency (GHz): 2.50
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: Intel(R) UHD Graphics 730
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 1024.00
	Graphics card #0 deviceId: 0x4692
	Graphics card #0 versionInfo: DriverVersion=31.0.101.3959
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.40
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.40
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 21532.91
	Virtual memory used (MB): 14084.59
	Swap memory total (MB): 5376.00
	Swap memory used (MB): 90.21
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		cloth-config: Cloth Config v9 9.0.94
			cloth-basic-math: cloth-basic-math 0.6.1
		enhancedblockentities: Enhanced Block Entities 0.7.2+1.19.2
			advanced_runtime_resource_pack: Runtime Resource Pack 0.6.2
			spruceui: SpruceUI 4.0.0+1.19
		fabric-api: Fabric API 0.68.1+1.19.3
			fabric-api-base: Fabric API Base 0.4.19+8d1895cf72
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.18+49abcf7e72
			fabric-biome-api-v1: Fabric Biome API (v1) 12.0.0+689f5e7172
			fabric-block-api-v1: Fabric Block API (v1) 1.0.3+12bfe4ea72
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.28+c6af733c72
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.9+49abcf7e72
			fabric-command-api-v1: Fabric Command API (v1) 1.2.19+f71b366f72
			fabric-command-api-v2: Fabric Command API (v2) 2.1.15+49abcf7e72
			fabric-commands-v0: Fabric Commands (v0) 0.2.36+df3654b372
			fabric-containers-v0: Fabric Containers (v0) 0.1.44+df3654b372
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.4.11+49abcf7e72
			fabric-convention-tags-v1: Fabric Convention Tags 1.1.9+2894d6df72
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.12+aeb40ebe72
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 11.0.1+65e415cb72
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.39+48349a3f72
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.5+b83334a072
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.37+422b77fb72
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.38+df3654b372
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.28+2894d6df72
			fabric-item-api-v1: Fabric Item API (v1) 2.1.4+fa25da9972
			fabric-item-group-api-v1: Fabric Item Group API (v1) 2.1.6+b7096da872
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.28+aaaf9d3372
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.26+df3654b372
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.8+2894d6df72
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.17+75e9821172
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.21+9e7660c672
			fabric-message-api-v1: Fabric Message API (v1) 5.0.12+6ede1da972
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.28+49abcf7e72
			fabric-models-v0: Fabric Models (v0) 0.3.25+11ba9c3b72
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.14+d9ac3c3972
			fabric-networking-v0: Fabric Networking (v0) 0.3.31+df3654b372
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 5.1.6+49abcf7e72
			fabric-particles-v1: Fabric Particles (v1) 1.0.18+fbde993d72
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.0.1+2608564672
			fabric-renderer-api-v1: Fabric Renderer API (v1) 2.1.0+2063beff72
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.23+11ba9c3b72
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.28+df3654b372
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.23+48349a3f72
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.16+3067f46372
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.31+df3654b372
			fabric-rendering-v1: Fabric Rendering (v1) 1.11.7+49abcf7e72
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.2.0+280be3ab72
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.10.2+f6c919d672
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.38+8c84281072
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.10+689f5e7172
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.6+75e9821172
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.11+28ac448f72
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 2.1.0+0efcd39b72
		fabric-language-kotlin: Fabric Language Kotlin 1.8.7+kotlin.1.7.22
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.7.22
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.7.22
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.7.22
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.7.22
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.18.5
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.4.1
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.4.1
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.4.1
		fabricloader: Fabric Loader 0.14.11
		gammautils: Gamma Utils 1.6.12
		indium: Indium 1.0.10+mc1.19.3
		iris: Iris 1.4.6
			io_github_douira_glsl-transformer: glsl-transformer 1.0.1
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.10.1
		java: OpenJDK 64-Bit Server VM 17
		lambdynlights: LambDynamicLights 2.1.2+1.19
			pride: Pride Lib 1.1.2+1.19
		lithium: Lithium 0.10.4
		minecraft: Minecraft 1.19.3
		modmenu: Mod Menu 5.0.2
		phosphor: Phosphor 0.8.1
		sodium: Sodium 0.4.6+build.20
		yet-another-config-lib: YetAnotherConfigLib 2.1.1
		zoomify: Zoomify 2.9.2
			com_akuleshov7_ktoml-core-jvm: ktoml-core-jvm 0.3.0
			com_github_llamalad7_mixinextras: mixinextras 0.1.1
			dev_isxander_settxi_settxi-core: settxi-core 2.10.6
			dev_isxander_settxi_settxi-gui: settxi-gui 2.10.6
			dev_isxander_settxi_settxi-kotlinx-serialization: settxi-kotlinx-serialization 2.10.6
			settxi-gui-yacl: Settxi Gui (YetAnotherConfigLib) 2.10.6
	Loaded Shaderpack: (off)
	NEC status: No NEC detected
	Launched Version: fabric-loader-0.14.11-1.19.3
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: Intel(R) UHD Graphics 730 GL version 3.2.0 - Build 31.0.101.3959, Intel
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: 12x 12th Gen Intel(R) Core(TM) i5-12400
#@!@# Game crashed! Crash report saved to: #@!@# C:\Users\Jack\AppData\Roaming\.minecraft\crash-reports\crash-2022-12-19_19.26.24-client.txt
19:26:25.367
Process crashed with exit code -1

**Reply by warjort** [reply]
Same issue, this time `enhanced block entities`.

You need to check that the mod author has actually tested the mod with the version of minecraft you are using.

Look at the "game version" on curseforge, e.g. enhanced block entities has no download marked for 1.19.3
https://www.curseforge.com/minecraft/mc-mods/enhanced-block-entities/files

**Comment by NBS-Paste** [comment]
I have all 1.19.3 mods and it loads into minecraft but it keeps showing the red mojang screen and it says 89 mods but i dont have 89 mods.

**Reply by haykam821** [reply]
The splash overlay is likely looping due to an issue with a mod during resource load. The log should show the error preventing resource load from completing.

Your mod count is inflated by mods that bundle other mods. For example, Fabric API is composed of many separate modules which are individually counted.

**Comment by NBS-Paste** [comment, marked as answer]
I’ve managed to fix it by now.

**Reply by TheKrafter** [reply]
could you share what you did? im experiencing a similar issue.

**Reply by NBS-Paste** [reply]
I reinstall fabric and that worked for me. I also was using a mod that didn’t work for 1.19.4 wich also caused it.
